spec/mailers/admin_mailer_spec.rb

```diff
@@ -81,6 +82,10 @@
       it("assigns @user_name") {
         mail_content(mail).should match(user.name)
       }
+      #it("assigns user.name in the message") {
```

Comment:
Code that will no longer be used can be removed

Reply:
already removed